en_US/data-integration/data-bridge-clickhouse.md

```diff
@@ -0,0 +1,150 @@
+# Clickhouse 
+
+EMQX supports integration with [Clickhouse](https://clickhouse.com/) so you can forward messages and events to Clickhouse. 
+
+{% emqxce %}
+:::tip
+The Clickhouse bridge is an EMQX Enterprise Edition feature. EMQX Enterprise Edition provides comprehensive coverage of key business scenarios, rich data integration, product-level reliability, and 24/7 global technical support. Experience the benefits of this [enterprise-ready MQTT messaging platform](https://www.emqx.com/en/try?product=enterprise) today.
+:::
+{% endemqxce %}
+
+::: tip Prerequisites
+
+- Knowledge about EMQX data integration [rules](./rules.md)
+
+- Knowledge about [data bridges](./data-bridges.md)
+
+- Basic knowledge about running commands in a UNIX terminal
```

Comment:
can we add a link here so they can know where to learn these knowledge?

Reply:
I think we can remove the word UNIX to make it sound less scary but I don't think a link will add much value. System administrators reading this document will know what a terminal is. So I think we can remove this line.